doc/performance-investigations/0012-increase-machine-type-core-count/summary.md

```diff
@@ -0,0 +1,49 @@
+# Investigate Machine Type Architecture Performance
+
+EQ Runner currently uses a machine type of `custom-4-4096` (4 vCPU cores, 4096MB of RAM). This investigation looked at the impact of machine type on the performance of the runner application, focusing on the number of CPU cores as Runner is primarily a CPU bound application with limited network IO to Datastore and Redis.
+
+The investigation aims to determine whether latencies as measured by the Locsut client improve when using larger machines but utilizing roughly the same percentage of CPU.
+
+## Benchmark profile
+
+| Option                 | Value                        |
+| ---------------------- | ---------------------------- |
+| Requests file          | census_household_gb_eng.json |
+| Run time               | 20m                          |
+| User wait time minimum | 1                            |
+| User wait time maximum | 2                            |
+| Clients                | 256                          |
+| Number of workers      | 7                            |
+| Number of threads      | 7                            |
+
+This test replicates the current settings for the daily test. Note that the baseline test ran with 64 clients as per the daily test.
+
+## Results
+
+| Cores | Pods | Nodes | GET Average | POST Average | All Request Average | Total Requests |
+| ----- | ---- | ----- | ----------- | ------------ | ------------------- | -------------- |
+| 4     | 4    | 4     | 572         | 609          | 590                 | 403,890        |
+| 8     | 4    | 2     | 542         | 579          | 560                 | 404,535        |
+| 8     | 4    | 2     | 591         | 601          | 596                 | 402,884        |
```

Comment:
What is different about these tests?

Reply:
Nothing, they were run twice so I've included both

Reply:
It makes it inconsistent, why was 8 cores run twice and not the others?

Reply:
Purely coincidental - the environment was up and ran at it's triggered time. Yes it's inconsistent, but it highlights how the figures are variable enough that we can glean very little info from the results.

Reply:
Maybe averaging them would be better then? If we don't think the results are reliable, then that brings into question the validity of the whole test.

Reply:
I think averaging just hides the variability. In terms of validity, I don't think the validity is a question, more that due to the variability of the tests, any improvement within the bounds of the variation can't be distinguished from the noise we see.

Reply:
I'm happy to remove the second run from the table and leave the data in the outputs